src/run.js

```diff
@@ -0,0 +1,79 @@
+const { fetchBinary, unpack } =  require("./download.js")
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+
+const RELAY_STARTUP_TIMEOUT = Number(process.env.RELAY_STARTUP_TIMEOUT || 30000)
+
+const run = async (configPath, identityPath) => {
```

Comment:
could we make both configPath and identityPath optional?

Reply:
Certainly.  Is this a reasonable default (pulled from ipfs test utils)?
```
{
  "RelayV1": {
    "Enabled": false
  },
  "RelayV2": {
    "Enabled": true
  },
  "Network": {
    "ListenAddrs": ["/ip4/127.0.0.1/tcp/24222/ws"],
    "AnnounceAddrs": ["/ip4/127.0.0.1/tcp/24222/ws"]
  },
  "Daemon": {
    "PprofPort": -1
  }
}
```

Reply:
I implemented the above, let me know if you'd like anything changed @ckousik.